RELEASE_PROCESS.md

```diff
@@ -0,0 +1,135 @@
+# Chef Server Release Process
+
+## Document Purpose
+
+The purpose of this document is to describe the *current* release
+process such that any member of the team can do a release.  As we
+improve the automation around the release process, the document should
+be updated such that it always has the exact steps required to release
+Chef Server.
+
+This document is NOT aspirational.  We have a number of automation
+tools that we intend to use to improve the release release process;
+however, many are not fully integrated with our process yet.. Do not
+add them to this document until they are ready to be part of the
+release process and can be used by any team member to perform a
+release.
+
+## Pre-requisites
+
+In order to release, you will need the following accounts/permissions:
+
+- Local checkouts of the chef-server and chef-web-downloads repositories
+- Push access to the chef-server github repository
+- Chef Software, Inc Hipchat account
```

Comment:
We've integrated Lita into Slack so promotions can now be performed in the #engineering-services channel in Slack.